src/lib/replication/repl_dev/raft_repl_dev.cpp

```diff
@@ -131,7 +131,7 @@ void RaftReplDev::async_alloc_write(sisl::blob const& header, sisl::blob const&
     } else {
         rreq->value_inlined = true;
         RD_LOG(DEBUG, "Skipping data channel send since value size is 0");
-        auto raft_status = m_state_machine->propose_to_raft(std::move(rreq));
+        auto raft_status = m_state_machine->propose_to_raft(rreq);
```

Comment:
I probably didn't get the reason for this change, since rreq is not const here, right?

Reply:
yes so that will make rreq into NULL and `handle_error` will panic when accessing this.

Reply:
ahh..., right.